### PR TITLE
feat(fe): Updated OrganizationDetailsEndpoint Urls to use Query Paramaters

### PR DESCRIPTION
**Pull request by iamrajjoshi**
In this pr, I update the urls for OrganizationDetailsEndpoint to use the new query parameter, `include_feature_flags`, which we will use to determine if the features data get returned in the API response object. 

We mentioned two places where we will be updating the urls in our research [here](https://www.notion.so/sentry/Remove-feature-flags-from-API-response-95ed20b0c6b74b7fad4cbc4495cd4c9b). The third place I modify is `preload-data.html` was because it was also preloading data and the preloaded org should also have all the features since they are used throughout our frontend.

Not sure if there are other gotchas for this.

### CODE REVIEW

**Review by ykamo001 — APPROVED** [review]
Thanks for making these changes

**Comment by ykamo001** [comment]
> we will use to determine if we are going to pass all the features
Would rephrase this to "we will use to determine if the features data get returned in the API response object

**Review by saponifi3d — APPROVED** [review]
